package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@balena/abstract-sql-compiler": "^9.0.4",
     "@balena/odata-to-abstract-sql": "^6.2.3",
+    "@balena/sbvr-types": "^7.0.6",
```

Comment:
This needs to be updated with the correct version once sbvr-types has a released version exposing typescript types, see https://github.com/balena-io-modules/sbvr-types/pull/98